en/Frequency-based.md

```diff
@@ -0,0 +1,14 @@
+## Practice Recommendations Organized by Use Case
+
+### Frequency-based trips
+
+A frequency-based trip does not follow a fixed schedule but attempts to maintain predetermined headways. These trips are denoted in [GTFS frequency.txt](https://gtfs.org/reference/static/#frequenciestxt) by setting `exact_times=0` or omitting the `exact_times` field. There are several best practices to keep in mind when constructing GTFS-realtime feeds for frequency-based trips.
+
+In [TripUpdate.StopTimeUpdate](#StopTimeUpdate), the [StopTimeEvent](#StopTimeEvent) for `arrival` and `departure` should not contain `delay` because frequency-based trips do not follow a fixed schedule. Instead, `time` should be provided to indicate arrival/departure predictions.
+
+Also in [TripUpdate.StopTimeUpdate](#StopTimeUpdate), `schedule_relationship` should not be `SCHEDULED` or `SKIPPED` because these values are only relevant for schedule-based trips (i.e., trips not defined in [frequencies.txt](https://gtfs.org/reference/static/#frequenciestxt) or trips defined in [frequencies.txt](https://gtfs.org/reference/static/#frequenciestxt) but with `exact_times=1`). As of May 2019, there is no enumeration value in`schedule_relationship` that is suitable for frequency-based trips. A new enumeration value, `UNSCHEDULED`, is being proposed to the official specification and will be the best option for frequency-based trips once approved. Until `UNSCHEDULED` is approved, we recommend leaving `schedule_relationship` empty.

```

Comment:
"UNSCHEDULED" has since been added. This section needs to be updated accordingly.

Reply:
I updated the text accordingly in https://github.com/MobilityData/GTFS_Realtime_Best-Practices/pull/1/commits/d2fccdf772ed4643d0a1e14e55383dbd661267e8